shared/scss/_mixins.scss

```diff
@@ -69,7 +69,8 @@
         position: absolute;
         right: 14px;
         width: 32px;
-        height: auto;
+        height: 22px;
+        margin-top: -1px;
```

Comment:
This makes the toggle appear vertically off-centre.

Try setting `top: 16px` on `.site-info__toggle-container` in `popup.scss` instead of doing the margin thing.

I have no idea why the toggle gets cut off without this. I played around with this and I'm stumped.

Reply:
@andrey-p I assume you're testing in Safari? I see the toggle vertically centered with the current styles. This was a fix for the cut off knob.
The weird thing is there was already a fix for it in the past, see [here](https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/51), but it occurred again while porting the latest changes.

Reply:
@MariagraziaAlastra I'm seeing this in Chrome/FF. It's pretty noticeable for me.

![screenshot from 2018-04-12 14-46-00](https://user-images.githubusercontent.com/1431577/38681350-56fcc9e6-3e60-11e8-823e-100bbbbffdfa.png)

Reply:
That fix also looks like it could do the job - it didn't make it into this PR since it was merged into the old branch, I guess.

Reply:
@andrey-p no, it's just the old fix didn't work anymore in this case :(

Reply:
Anyway thanks! The cut off knob fix in this PR is actually mine - Tried to fix it in the Safari browser inspector - so I take the blame for it messing with the vertical alignment in FF/Chrome 😄

Reply:
Changing the margin on `.site-info__toggle-container` doesn't fix it.